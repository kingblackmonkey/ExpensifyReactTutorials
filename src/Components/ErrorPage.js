import React from 'react';
import {Link} from 'react-router-dom';
const ErrorPage = ()=>{
    return (
        <div>
            <p>404!! not found</p>
            <Link to="/">Go Home</Link>
        </div>
        
  
    )
  }

  export default ErrorPage ; 