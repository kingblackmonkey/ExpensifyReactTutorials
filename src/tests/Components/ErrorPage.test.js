import React from 'react';
import { shallow} from 'enzyme';
import ErrorPage from '../../Components/ErrorPage'

test (' should render Error component', ()=>{
    const wrapper = shallow(<ErrorPage />)
    expect (wrapper).toMatchSnapshot()
})