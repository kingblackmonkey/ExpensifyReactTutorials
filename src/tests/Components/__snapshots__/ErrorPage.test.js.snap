// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render Error component 1`] = `
<div>
  <p>
    404!! not found
  </p>
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
